webapp/portlet/src/main/webapp/idm-groups-management/components/GroupsManagementFormDrawer.vue

```diff
@@ -161,6 +161,15 @@ export default {
       this.drawer = true;
     },
     saveGroup(event) {
+      const regex =  /^[a-zA-Z0-9-_]+$/;
+      const isValid = regex.test(this.group.groupName);
+      if (!isValid){
+        this.error = this.$t('GroupsManagement.popup.message');
```

Comment:
you can call this.handleError(this.$t('GroupsManagement.popup.message')) instead.
+
`GroupsManagement.popup.message` is not a significant key, you can use `GroupsManagement.error.invalidGroupName` for example.

Reply:
done